sentry/src/routes/channel.rs

```diff
@@ -596,4 +629,105 @@ mod test {
             assert_eq!(expected, res.expect_err("Should return an error"));
         }
     }
+
+    #[tokio::test]
+    async fn adds_and_retrieves_spender_leaf() {
+        let app = setup_dummy_app().await;
+        let channel = DUMMY_CAMPAIGN.channel;
+
+        insert_channel(&app.pool, channel)
+            .await
+            .expect("should insert channel");
+
+        let build_request = |channel: Channel| {
+            Request::builder()
+                .extension(channel)
+                .body(Body::empty())
+                .expect("Should build Request")
+        };
+        let build_request_with_params = |channel: Channel| {
```

Comment:
Since `build_request` & `build_request_with_params` are used to build 2 requests for 2 different routes, I would suggest we rename them to something less generic as a name.

`add_spender_request`
`get_accounting_request`